.github/workflows/run-tests.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       fail-fast: true
       matrix:
         os: [ubuntu-latest, windows-latest]
-        php: [8.0]
+        php: [7.4, '8.0', 8.1]
```

Comment:
Added PHP 7.4 support for people still stuck to older versions. I'm not 100% sure (didn't test it) but I think you can even install it on 7.3 but I would not suggest to add it as its already out of the maintainance and security support.

Others can use e.g. RectorPHP to generate a <7.4 compatible version of the library